src/docs/Ways of Working/percy.md

```diff
@@ -0,0 +1,166 @@
+
+// A script to navigate our app and take snapshots with Percy.
+PercyScript.run(async (page, percySnapshot) => {
+  let server = httpServer.createServer();
+  server.listen(PORT);
+
+  console.log(`Server started at ${TEST_URL}`);
+
+  await page.goto(TEST_URL);
+  await percySnapshot('index',{ widths: [375, 767, 1280] });
+
+  server.close();
+});
+
+```	
+
+5. Create npm script to run test
+    - Add the following in your `package.json` file under `scripts`
+
+```js
+"test-percy": "percy exec -- node tests/scripts/percy.js"
+```
+
+6. Test your script from the `command line`
+    - You first need to set the enviromental variable `PERCY_TOKEN`. For example on Windows `set PERCY_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX`
+    - Then simply run `npm run test-percy`
+    - If all goes well a link will appear in the command line where you can view the screenshoots.
+
+7. Create a github workflow 
+    - Create a workflow file under `.github/workflows/percy.yml`. See below a sample workflow which fires on pull requests and push on main branch.
+
+```yml
+name: Percy CI
+on: 
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+   branches:
+     - main
+  pull_request:
+    branches:
+     - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Install
+        run: npm install
+      - name: Percy Test
+        uses: percy/exec-action@master
+        with:
+          command: "node tests/scripts/percy.js"
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
+
+```
+
+## Run percy from command line
+
+From the `command line`
+- You first need to set the enviromental variable `PERCY_TOKEN`. For example on Windows `set PERCY_TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX`
+- Then simply run `npm run test-percy`
+- If all goes well a link will appear in the command line where you can view the screenshoots. 
+
+## Github review builds
+
+Once you've added the app and linked projects with repositories, subsequent Percy builds will show metadata from GitHub and Percy will automatically update the pull request summary if there are visual differences waiting for review.
+
+![Github pull request review](../assets/percy6.png)
+
+*Note: By default, Percy approvals aren’t required before merging, but you can enable this feature once you’re ready for it.*
+
+Clicking "Details" will take you directly to the Percy build UI showing all visual diffs to be reviewed. 
+  	
+![Percy snapshoots page](../assets/percy5.png)
+  
+Approving a build in Percy sets the Percy status in GitHub to green/successful, so that your team can see that any visual changes were reviewed and approved.
+
+![Github pull request review](../assets/percy7.png)
+
+### Adding comments
+
+To add a comment and start a comment thread, select the icon in the snapshot header.
+
+![Percy comments](../assets/percy8.png)
+
```

Comment:
You have to select request changes to start comments right? I wasn't sure if this was the right image because the image says 'request changes'

Reply:
Had the wrong image there sorry... corrected it

Reply:
Nice 👍